docs/content/getting-started.mdx

```diff
@@ -4,9 +4,11 @@ title: "Welcome to Dagster! | Dagster Docs"
 
 # Welcome to Dagster!
 
-<p className="text-2xl mt-0 text-gray-500 tracking-tight font-light">
-  Dagster is the data orchestration platform built for productivity.
-</p>
+Dagster is an orchestrator that's designed for developing and maintaining data assets, such as tables, data sets, machine learning models, and reports.
+
+You declare functions that you want to run and the data assets that those functions produce or update. Dagster then helps you run your functions at the right time and keep your assets up-to-date.
+
+Dagster is built to be used at every stage of the data development lifecycle - local development, unit tests, integration tests, staging environments, all the way up to production.
 
 Dagster is an orchestrator designed for developing and maintaining data assets, such as tables, data sets, machine learning models, and reports.
 
```

Comment:
This guy is a dupe and needs to come out

Reply:
Oops thanks for catching! Fixed.